apps/block_scout_web/lib/block_scout_web/etherscan.ex

```diff
@@ -871,6 +871,84 @@ defmodule BlockScoutWeb.Etherscan do
     "result" => []
   }
 
+  @epoch_getvoterrewards_example_value %{
```

Comment:
Should we then remove/deprecate the existing API methods from the `reward` module?
![image](https://user-images.githubusercontent.com/20768968/197596636-af222870-9c6e-4df6-b3a2-103de1df0775.png)

Reply:
Planning to do that once we have all endpoints implemented.